easypost/services/end_shipper_service.py

```diff
@@ -0,0 +1,49 @@
+from typing import (
+    List,
+    Optional,
+)
+
+from easypost.easypost_object import convert_to_easypost_object
+from easypost.models.address import Address
+from easypost.models.end_shipper import EndShipper
+from easypost.requestor import (
+    RequestMethod,
+    Requestor,
+)
+from easypost.services.base_service import BaseService
+
+
+class EndShipperService(BaseService):
+    def __init__(self, client):
+        self._client = client
+        self._model_class = Address.__name__
+        self._service_class = EndShipper.__name__
+
+    def create(self, api_key: Optional[str] = None, **params) -> Address:
+        """Create an EndShipper."""
+        url = self._class_url(self._service_class)
+        wrapped_params = {self._snakecase_name(self._model_class): params}
+
+        response, api_key = Requestor(self._client).request(method=RequestMethod.POST, url=url, params=wrapped_params)
+
+        return convert_to_easypost_object(response=response, api_key=api_key)
+
+    def all(self, **params) -> List[Address]:
+        """Retrieve a list of EndShippers."""
+        return self._all_resources(self._service_class, **params)
+
+    def retrieve(self, id) -> Address:
```

Comment:
```suggestion
    def retrieve(self, id: str) -> Address:
```

This type hint is also missing in other services retrieve function

Reply:
As mentioned elsewhere, I will come back and do type hints separately